src/styles/blocks/page.css

```diff
@@ -0,0 +1,11 @@
+.page {
+  font-family: "Roboto", Helvetica, sans-serif;
```

Comment:
move to html selector

Reply:
.page is a selector html class. 
I did as in the BEM lecture.